instant.ts

```diff
@@ -202,12 +259,19 @@ const main = async () => {
           name: 'dev',
           alias: 'd',
           type: Boolean
+        },
+        {
+          name: 'concurrency',
+          alias: 'c',
+          type: Number,
+          defaultValue: 5
```

Comment:
```suggestion
          defaultValue: 5
```
Is this something that could be configureable using a env var ?

Reply:
It's not, but I don't think it needs to be as the rest of the commandline options don't work that way. It should be exposed as a command line option from the instant cli. (i.e. `instant package init -c 10 -p cdr --dev`) This is a step towards that but we still need changes to the go CLI to fully enable it.